src/box/index.cc

```diff
@@ -457,23 +458,58 @@ box_index_iterator(uint32_t space_id, uint32_t index_id, int type,
 	uint32_t part_count = mp_decode_array(&key);
 	if (key_validate(index->def, itype, key, part_count))
 		return NULL;
+	const char *pos, *pos_end;
+	char *buf = NULL;
```

Comment:
Nit: Please rename to `pos_buf` - there may be other buffers in this function.

Reply:
Done